skxray/fitting/lineshapes.py

```diff
@@ -353,9 +353,8 @@ def compton(x, compton_amplitude, coherent_sct_energy,
 
     x = e_offset + x * e_linear + x**2 * e_quadratic
 
-    compton_e = (coherent_sct_energy
-                 / (1 + (coherent_sct_energy / 511)
-                    * (1 - np.cos(compton_angle * np.pi / 180))))
+    compton_e = coherent_sct_energy / (1 + (coherent_sct_energy / 511) *
```

Comment:
This is mc2, the rest-mass energy of an electron (511 keV). I will put more explanation. Please refer to http://en.wikipedia.org/wiki/Compton_edge for more information.